next.config.js

```diff
@@ -3,6 +3,7 @@ const nextra = require("nextra");
 const nextraDocsConfig = {
   theme: "nextra-theme-docs",
   themeConfig: "./theme.config.jsx",
+  latex: true
```

Comment:
I would expect this to fuck up the `$` we use inside of some of the docs pages (like on our env var page) but seemingly it doesn't...? wacky.